addons/bus/static/src/debug/bus_logs_menu_item.js

```diff
@@ -0,0 +1,39 @@
+import { Component, useRef, useState } from "@odoo/owl";
+import { DropdownItem } from "@web/core/dropdown/dropdown_item";
+import { registry } from "@web/core/registry";
+import { useService } from "@web/core/utils/hooks";
+
+/**
+ * @typedef {Object} Props
+ * @extends {Component<Props, Env>}
+ */
+export class BusLogsMenuItem extends Component {
+    static template = "bus.BusLogsMenuItem";
+    static components = { DropdownItem };
+
+    setup() {
+        this.busLogsService = useState(useService("bus.logs_service"));
+        this.downloadButton = useRef("downloadButton");
+    }
+
+    onClick(ev) {
+        if (this.downloadButton.el?.contains(ev.target)) {
+            this.env.services.bus_service.downloadLogs();
+        } else {
+            this.busLogsService.toggleLogging();
+            ev.preventDefault();
+            ev.stopPropagation();
```

Comment:
Why do we need those? And if we need them, why only in the `else`?

Reply:
I feel like toggling logging on/off should not close the menu (preventDefault). And clicking on the label should result in 1 click action, not one for the label and one for the outer div that holds the listener (stopPropagation).

Reply:
Understood